### DISCUSSION TITLE
[Careers] 예진의 2023 게임 인턴십 (2월, 5월)

### DISCUSSION THREAD

**Discussion by pnudpwls1103** [BDD-CLUB/challenge-every-week-posting#48] [4. 기타 글 모음]
24학점 들으면서 서류를 준비하기는 쉽지 않았습니다.
5월은 놀기도 했지만, 후반으로 갈수록 서류 마감에 찌들었던 거 같네요.
(일 벌여서 미리 준비 못 한 제 잘못이기도 하죠 ^^)
아쉽게 서류도 통과 못한 결과가 많이 있지만 인턴보다 더 괜찮은 결과가 들어와서 다음 글은 이력서 및 면접 후기를 들고 올 예정입니다. 또한, 구직 관련 앱 추천도 가져올 예정입니다.

6월까지는 취업 관련 얘기를 적고 7월부터는 기술 관련 글을 적을 수 있길..ㅎㅎ

[블로그 링크](https://donkeysdevelpment.tistory.com/114)

**Comment by publdaze** [comment]
우와..! 바쁜 와중에도 꾸준히 취업 준비하는 거 대단합니다!! 화이팅화이팅이에요!
인턴보다 더 괜찮은 결과라니! 기대됩니다! 🤩🤩

**Reply by pnudpwls1103** [reply]
관심 감사합니다 ㅎㅎㅎ

**Comment by shkisme** [comment]
24학점과 취준을 병행하는게 가능한 일이였군요... 대단... 정말로 고생 많으셨어요...!!
다음 글도 기다리겠습니다 🤩🤩

**Reply by pnudpwls1103** [reply]
감사합니다 ❤❤
